src/util/wren2c.c

```diff
@@ -0,0 +1,81 @@
+
+internal 
+void WREN2C_randomString(char *dest) {
+    // https://stackoverflow.com/a/15768317
+    char charset[] = "0123456789"
+                     "abcdefghijklmnopqrstuvwxyz"
+                     "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    srand(time(0));
+
+    size_t length = 10;
+    while (length-- > 0) {
+        size_t index = (double) rand() / RAND_MAX * (sizeof charset - 1);
+        *dest++ = charset[index];
+    }
+    *dest = '\0';
+}
+
+
+internal 
+void WREN2C_encodeAndDump(ENGINE* engine, int argc, char* args[]) {
+  
+  char* fileName = args[2];
+  char* fileOut = strcat(strdup(fileName), ".inc");
+
+  char randName[] = "";
+  WREN2C_randomString(randName);
```

Comment:
Why are you using a random name here? It would be better to allow a consistent and user-set module/variable name so this can be incorporated into build tools.